birdhouse/config/finch/docker-compose-extra.yml

```diff
@@ -0,0 +1,26 @@
+version: "3.4"
+
+x-logging:
+  &default-logging
+  driver: "json-file"
+  options:
+    max-size: "50m"
+    max-file: "10"
+
+services:
+  finch:
+    image: ${FINCH_IMAGE}
+    container_name: finch
+    environment:
+      HOSTNAME: $HOSTNAME
+      HTTP_PORT: 5000
+      PYWPS_CFG: /wps.cfg
+    ports:
+      - "8095:5000"
+    volumes:
+      - ./config/finch/wps.cfg:/wps.cfg
+      # - data:/opt/birdhouse/var/lib
+      - wps_outputs:/data/wpsoutputs
```

Comment:
The new `wps_outputs-volume` config/component is still volume-mounted in each component directly (still ok from a technical point).
I'm wondering if it would make sense to have them separated in their own 
`birdhouse/compose/finch/config/wps-outputs/docker-compose-extra.yml` file, just to be consistent with other component dependencies structure?

Reply:
I could do that.... that would imply that the wps_outputs volume is optional for these components which might actually work.... let me think about this a little bit.

Reply:
It wouldn't make much sense to have the birds run without WPS outputs as there would be no way to download the resulting outputs from the processing (they don't serve the output URL by themselves).

I don't think we must necessarily imply "components = optional". For example, Cowbird adds Magpie-specific configs when enabled, but Magpie is required when using Cowbird, otherwise it doesn't really have anything to work with.

In this case, I wouldn't mind that much if the volume definition remained in this file since the WPS output location and the volume location is highly coupled with the values configured in its `wps.cfg`, and the URL to the nginx `proxy` location.

Reply:
Ok that makes sense to me.